src/libstd/fs.rs

```diff
@@ -422,6 +422,11 @@ impl OpenOptions {
     /// This option, when true, means that writes will append to a file instead
     /// of overwriting previous contents.
     ///
+    /// Note that this isn't the same as `write` with the cursor starting at
+    /// the end of the file; this mode only allows writing to the end of the
```

Comment:
@steveklabnik if I understand what is going on, it seems like there must be some difference between writing and appending as a file operation, and this is what `cat` would be using in your example. This is news to me, if it is the case; is there a method in Rust somewhere that implements this append operation that I was missing?